lstchain/calib/camera/pixel_threshold_estimation.py

```diff
@@ -79,3 +83,78 @@ def get_unusable_pixels(dl1_path, pedestal_id):
                                                       HIGH_GAIN,
                                                       :] == True)
     return unusable_pixels
+
+
+def find_safe_threshold_from_dl1_file(dl1_path, config_file=None,
```

Comment:
Wouldnt it be simpler to directly give the parameters to this function and avoid the standard config dance below?

Reply:
I think that this will make life easier to lstosa and lstmcpipe